docs/cvdr.md

```diff
@@ -0,0 +1,112 @@
+# Use cvdr from your local machine
+
+This page describes how to run `cvdr` to use cloud orchestrator remotely.
+
+## Build cvdr
+
+Please run:
+```bash
+cd /path/to/cloud-android-orchestration
+go build ./cmd/cvdr
+```
+
+## Find URL of cloud orchestrator
+
+If cloud orchestrator manages docker instances, please read
+[docker.md](docker.md) to check URL of cloud orchestrator.
```

Comment:
We may also describe how to get the `SERVICE_URL` when the CO is not using docker IM. Is it `http://${CLOUD_ORCHESTRATOR_IP_ADDRESS}`?

Reply:
It would be, but there's no external use cases of `GCEIMType` currently AFAIK.

For local cases, the service url should be `http://localhost:8080`.

Reply:
BTW, I think this is out of scope from this change. I think the best way for managing documents would be uploading document for docker case first, and updating it including `HttpBasicAM` or `LocalIM`.

Reply:
SGTM!